examples/mariadb-claim.yaml

```diff
@@ -7,7 +7,7 @@ spec:
   parameters:
     region: us-west-2
     engine: mariadb
-    engineVersion: "10.6.10"
+    engineVersion: "10.6.19"
```

Comment:
wonder if we can have these versions in sync 

https://github.com/upbound/configuration-aws-database/blob/main/examples/mariadb-claim.yaml#L10
an default in the XRD in configuration-aws-database - so we get rid of bumping this in 2 areas - wdyt ?

Reply:
Tempting idea but it will create a slight bias towards mariadb level on xrd-level as we also will need to set the engine default and tbh, i actually like that we need to be specific about this atm. Another problem is that it will only solve the problem for mariadb but not for postgres or other engines we might support in the future.
What i liked more was your idea of a renovate plugin. 
Another idea i'm thinking of is, how do we generate these examples? Maybe samples generation could introduce a centralized layer for mapping such things 🤔

Reply:
right missed that we have the other variants too ;( yea lets go the renovate way